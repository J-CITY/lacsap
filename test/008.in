and 
array
begin
break
case
const
continue
div
do
downto
else
end
false
file
for
function
goto
if
lable
mod
nil
not
of
or
packed
procedure
program
record
repeat
set
shl
shr
string
then
to
true
type
unit
until
var
while
with
xor
