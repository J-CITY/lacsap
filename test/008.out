ERROR: -1006 6 56
